index.html

```diff
@@ -294,7 +294,7 @@ <h2 class="header-primary-text">contact me</h2>
                 built or a project that
                 needs
                 coding. I’d love to help with it</p>
```

Comment:
Please add the project description under the project name in the README file so that users can know what is this project about.